.github/lpteams/lp-teams-check.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+cd "$(dirname "$0")" || exit
+
+if ls -- *.team 1>/dev/null 2>&1; then
+    readarray -t TEAMS < <(basename --multiple --suffix=.team -- *.team)
+else
+    exit
+fi
+
+for LP_TEAM in "${TEAMS[@]}"; do
+    EXISTING_TEAM=$(cat "$LP_TEAM.team")
+    READ_TEAM=$(curl --silent https://api.launchpad.net/1.0/~"${LP_TEAM}"/members |
```

Comment:
I believe this should be `https://api.launchpad.net/devel/...`: Launchpad `1.0` is deprecated as attempts to version the API were abandoned over a decade ago, AIUI. This was documented somewhere, perhaps https://help.launchpad.net/API/ - but the deep links have not been preserved (the irony!) so I cannot immediately find it.

Reply:
You're right, I confirmed with the LP team. Thanks.